### PR TITLE
Jetpack monitor verbiage update. changing USA centric idiom

### PR DESCRIPTION
**Pull request by jeffgolenski**
Jetpack monitor descriptive text in settings. Removing a USA-centric idiom, "keep tabs on."  Replaced with "Continuously watches" on both the Jetpack "at a glance" Dashboard and Jetpack security settings within wp-admin

Fixes https://github.com/Automattic/jetpack-com/issues/77

Before:
image
screen shot 2018-12-19 at 10 33 20 am

After:
screen shot 2018-12-19 at 10 30 30 am
screen shot 2018-12-19 at 1 41 42 pm

#### Proposed changelog entry for your changes:

* no proposed changlog entry. Just a minor text update

Testing instructions:

1) Sync up with this branch
2) Navigate to security settings. Find Monitor card
3) Click / tap the i icon 
4) Navigate to jetpack dashboard
5) Click / tap the i icon on the monitor card

### CODE REVIEW

File: _inc/client/security/monitor.jsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const Monitor = withModuleSettingsFormHelpers(
 						disableInDevMode
 						module={ this.props.getModule( 'monitor' ) }
 						support={ {
-							text: __( 'Keep tabs on your site and receive alerts the moment downtime is detected.' ),
+							text: __( 'We will contiously watch your site and receive alerts the moment downtime is detected.' ),
```
Comment (jeherve) [inline _inc/client/security/monitor.jsx]: I think you meant "continuously" maybe?

**Review by jeherve — CHANGES_REQUESTED** [review]
This looks good, apart from a typo with continuously I think.

**Comment by jeffgolenski** [comment]
@jeherve good find. I was just having some difficultly with git and this brand, but sorted it out (don't mind all the commits). Can you give it another review?

File: _inc/client/security/monitor.jsx [hunk 2]
```diff
@@ -35,9 +35,7 @@ export const Monitor = withModuleSettingsFormHelpers(
 						disableInDevMode
 						module={ this.props.getModule( 'monitor' ) }
 						support={ {
-							text: __(
-								'Keep tabs on your site and receive alerts the moment downtime is detected.'
-							),
+							text: __( 'We will continuously watch your site and receive alerts the moment downtime is detected.' ),
```
Comment (jeherve) [inline _inc/client/security/monitor.jsx]: Should it say "and you will"? Right now it sounds like "we" will receive the alerts.

**Review by jeherve — CHANGES_REQUESTED** [review]
Sorry, I'm nit-picking again.

**Comment by jeffgolenski** [comment]
thanks. I'm having trouble with git right nowI thought I updated that

**Comment by jeffgolenski** [comment]
Alright. @jeherve I think we're good. Can you give this another review?

**Review by jeherve — APPROVED** [review]
Looking good now! 🚢

**Pull request merged**